### PR TITLE
Update: Custom dictionary style from web style

### PR DESCRIPTION
**Pull request by yo1995**
This PR adds the capability of creating a custom dictionary style from an AGOL hosted web style. Details in `common-samples/issues/2327`

### CODE REVIEW

File: arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md [hunk 1]
```diff
@@ -10,27 +10,35 @@ When symbolizing geoelements in your map, you may need to convey several pieces
...
 
-Pan and zoom the map to see the symbology from the custom dictionary style.
+Toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style. The web style and style file are slightly different to each other to give a visual indication of the switch between the two.
 
 ## How it works
 
-1. Create a new `AGSDictionarySymbolStyle` by passing in the URL to the custom style file (.stylx).
-2. Create a new `AGSDictionaryRenderer`, providing the dictionary symbol style.
-3. Apply the dictionary renderer to a feature layer.
+1. Create an `AGSPortalItem`, referring to a `AGSPortal` and the item ID of the web style.
```
Comment (vquach2404) [inline arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md:17]:
```suggestion
1. Create an `AGSPortalItem`, referring to an `AGSPortal` and the item ID of the web style.
```

**Review by vquach2404 — APPROVED** [review]
LGTM! Just one change needed

File: arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md [hunk 2]
```diff
@@ -10,27 +10,35 @@ When symbolizing geoelements in your map, you may need to convey several pieces
...
+Toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style. The web style and style file are slightly different to each other to give a visual indication of the switch between the two.
 
 ## How it works
 
-1. Create a new `AGSDictionarySymbolStyle` by passing in the URL to the custom style file (.stylx).
-2. Create a new `AGSDictionaryRenderer`, providing the dictionary symbol style.
-3. Apply the dictionary renderer to a feature layer.
+1. Create an `AGSPortalItem`, referring to an `AGSPortal` and the item ID of the web style.
+2. Based on the style selected:
+    * Web style: create a new `AGSDictionarySymbolStyle` from the portal item using `AGSDictionarySymbolStyle.init(portalItem:)` with the portal item.
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md:19]:
```suggestion
    * Web style: create a new `AGSDictionarySymbolStyle` from the portal item using `AGSDictionarySymbolStyle.init(portalItem:)`.
```

**Comment by yo1995** [comment]
From Friday's review

- Change the segmentedControl text to local file or local stylx file
- Change the initial viewpoint to Esri campus and zoom scale more detailed

**Comment by yo1995** [comment]
From Friday: Add a field remapping to the dictionary renderer

File: arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md [hunk 3]
```diff
@@ -10,27 +10,35 @@ When symbolizing geoelements in your map, you may need to convey several pieces
...
+The data used in this sample is from a feature layer showing a subset of [restaurants in Redlands, CA](https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer) hosted as a feature service with attributes for rating, style, health score, and open hours.
+
+The feature layer is symbolized using a dictionary renderer that displays a single symbol for all of these variables. The renderer uses symbols from a custom restaurant dictionary style created from a [stylx file](https://arcgis.com/home/item.html?id=751138a2e0844e06853522d54103222a) and a [web style](https://arcgis.com/home/item.html?id=adee951477014ec68d7cf0ea0579c800), available as an items from ArcGIS Online, to show unique symbols based on several feature attributes. The symbols it contains were created using ArcGIS Pro. The logic used to apply the symbols comes from an Arcade script embedded in the stylx file (which is a SQLite database), along with a JSON string that defines expected attribute names and configuration properties.
```
[the remark this answers is not in the document]
Comment (yo1995) [reply arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md:36]: The second is correct - it's being fixed in the design!

**Review by saratchandrakarumuri — COMMENTED** [review]
I see the updated readme file, but within sample app, info page remains unchanged. I have deleted and clean reinstalled it but still see the same behavior

IMG_0158
IMG_0159

**Review by saratchandrakarumuri — APPROVED** [review]
New changes looks good

**Pull request merged**